packages/create-pantheon-decoupled-kit/package.json

```diff
@@ -18,7 +18,6 @@
 		"npm init"
 	],
 	"type": "module",
-	"main": "/dist/bin.js",
```

Comment:
This was causing `pnpm link` to fail (and also it comes up as a warning in publint.dev)